sandpack-themes/src/danielAwesomePro.ts

```diff
@@ -0,0 +1,36 @@
+import type { SandpackTheme } from "./types";
+
+export const danielAwesomePro: SandpackTheme = {

```

Comment:
Hey @D-Lite, thanks for your awesome contribution! 

Your theme looks good, but I just want to point one tweak here. Every Sandpack Theme usually have a name that represents its styling or a proper name, examples: `githubLight`, `atomDark`, etc. 

So, in order to go further and merging it, I'd like to ask to rename it to something that represents the theme and not the author name or generic names such as `awesome` and `pro` that don't represent very well the theme itself.

Reply:
Hi @danilowoz 

This has been resolved and I just made a new commit!

Reply:
I think you forgot to push your changes, because I can't see it on the diff

Reply:
@danilowoz, I had to push and make changes manually

Now you can see the diff.